### ISSUE TITLE
Please document any force pushes to master branch

### ISSUE THREAD

**Issue by sidkshatriya** [elixir-lang/elixir#12573]
### Elixir and Erlang/OTP versions

N/A

### Operating system

N/A

### Current behavior

I tried to `git pull origin main` just now. I notice that the `main` branch has been force pushed too (git warns me about a merge issue) . 

I can verify that there are commits which I had locally which are no longer in `main` branch. This is puzzling. 

### Expected behavior

I can realize it may be occasionally necessary to force push to the `main` branch -- can it just be documented somewhere that it was done?

**Comment by josevalim** [comment]
That's a good suggestion. Do you have any ideas on how/where we should document it? We could send an email to elixir-lang-core but I am not sure if that's enough. Thoughts?

**Comment by sidkshatriya** [comment]
For instance c0726e88133b0dd16613456a458be5944f3c7cda used to exist as far as I can tell in `main` but now it doesn't. It has become a107d6d0f5998b8f332c265083e3cc59033c84b8 .

BTW Just for my understanding why was this force push done?

----

As a start, I think we should just create a github issue to document every force push to main. We can title it `[Force Push] <Brief Reason>`. Provide any details in the issue if necessary. People trying to figure out what happen would probably first come to the issue tracker to search for various keywords -- "force push" is what I tried to search but nothing came up...

Its best for this information to always be in github (in addition to anywhere else) because everybody may not be aware of or subscribed to `elixir-lang-core` etc.

In fact I ran into a somewhat related issue earlier. When the repo moved from using `master` to `main` it would have been nice to just have a tracking issue to briefly document that the repo was moving to `main` as default branch. I was puzzled briefly on that also on the git command line. Of course that was a bit more obvious because the default branch showing was `main` when you came to the repo on github. 

TL;DR It would be great if any forced change to HEAD is briefly documented as a github issue.

**Comment by josevalim** [comment]
done here: https://github.com/elixir-lang/elixir/issues/12575

we will attempt to do this from now on, thanks for the suggestion! Next time we will have more information too, but I no longer have the exact SHA.

**Comment by sidkshatriya** [comment]
Thanks so much for your quick response ! :-)
